README.md

```diff
@@ -1,32 +1,122 @@
-# Chained Conditional
+# Chained Conditional Statements
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Continue learning about conditionals.
+- Introduce chained conditional statements.
 
 ## Introduction
 
-A 1-2 sentence summary of what will be covered.
+In the previous lesson we introduced the `if-else` statement. In this lesson, we
+will continue expanding on the topic of conditionals but introducing a chain of
+conditional statements.
 
-## Topic 1
+Let's look at the `else if` statement.
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+## Else If Statements
 
-## Topic 2
+An `if` statement can also be followed by an `else if` statement. When an
+`else if` statement follows an `if`, the `else if` allows us to specify
+another condition whenever the conditional clause of the `if` statement is
+not true. It is similar to an `else` in that it is not required to follow
```

Comment:
Similar to my comment on the other PR, this sentence sounds a little ambiguous.
Consider 'It is similar to an `else` in that an `if` does not need to be followed by an `else if`'

Reply:
Perhaps also mention that the `else` is not required